extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs

```diff
@@ -3,4 +3,4 @@
 
 ﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.ServiceBus", "5.0.0-beta.6")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.ServiceBus", "5.3.0")]
```

Comment:
Looks like 5.4.o is the latest version of this package. https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/

Reply:
It was released yesterday 😆

Reply:
Good catch, I'll update this.